### PR TITLE
i18n(fr): Update `install-and-setup.mdx` from #8942

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `install-and-setup.mdx` from #8942

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/install-and-setup.mdx [hunk 1]
```diff
@@ -22,12 +22,15 @@ Vous préférez essayer Astro dans votre navigateur ? Consultez [astro.new](ht
 - **Éditeur de texte** - Nous recommandons [VS Code](https://code.visualstudio.com/) avec notre [extension officielle Astro](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
 - **Terminal** - Astro est accessible via son interface en ligne de commande (CLI).
 
+## Compatibilité avec les navigateurs
+
+Astro est construit avec Vite, qui cible par défaut les navigateurs supportant le JavaScript moderne. Pour une référence complète, vous pouvez consulter la [liste des versions de navigateurs actuellement prises en charge par Vite](https://vitejs.dev/guide/build.html#browser-compatibility).
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/install-and-setup.mdx:27]:
```suggestion
Astro est construit avec Vite, qui cible par défaut les navigateurs supportant le JavaScript moderne. Pour une référence complète, vous pouvez consulter la [liste des versions de navigateurs actuellement prises en charge par Vite](https://vitejs.dev/guide/build.html#browser-compatibility).

```

**Review by ArmandPhilippot — APPROVED** [review]
LGTM! (I just suggested a new line after the new section)

**Review by ArmandPhilippot — APPROVED** [review]
LGTM, thanks @thomasbnt!

**Review by yanthomasdev — APPROVED** [review]
LGTM, thanks!

**Pull request merged**